doc/linux/SELINUX.md

```diff
@@ -0,0 +1,46 @@
+# Dinit SELinux Awareness
+
+Dinit has support for basic SELinux awareness. This document is intended to
+outline the extent and inner workings of Dinit's SELinux awareness. The reader
+is assumed to be knowledgeable about the basics of [SELinux](https://github.com/SELinuxProject/selinux-notebook) and Dinit.
+
+Dinit needs to be built with SELinux support (see [BUILD](/BUILD)) to enable the features that are
+mentioned in this document.
+
+## Loading the system SELinux policy
+When booted as the system init system, dinit by default will attempt to load the
+system's SELinux policy and transition itself to a context specified by that policy
+if not already done so in earlier boot (e.g. by an initramfs). This behaviour may be
+disabled by passing dinit the `--disable-selinux-policy` flag. As dinit will always
+be PID1 in this senario, this can be done by appending the flag to the kernel cmdline.
```

Comment:
Not 'cmdline', it's 'command line'.

Whether an argument appended to the kernel command line is passed on to dinit is going to be dependent on the initrd, if there is one; this statement needs to be corrected.

Reply:
As passing options to dinit is already documented better in other places I've just removed that statement all together, is that okay?

Reply:
Yes, that's ok.